### DISCUSSION TITLE
VersionedTransaction

### DISCUSSION THREAD

**Discussion by celliott412** [bmresearch/Solnet#455] [Ideas]
I would like to use the Jupiter V6 Swap API with Solnet, however, it looks like Solnet doesn't have VersionedTransaction implemented. This class is pivotal for deserializing and signing the transaction.

Is there any intention to add VersionedTransaction to Solnet?

**Comment by shihanhana** [comment]
Same question.

**Comment by 1337Programmer** [comment]
Would like to know about this as well!
